### PR TITLE
[DCS] Change Capacity from int to float64

### PR DESCRIPTION
**Pull request by Aloento**
## Summary of the Pull Request

Change Capacity from int to float32

## PR Checklist

* [x] Refers to: https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1839
* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.
* [x] Release notes added.

## Acceptance Steps Performed

```
=== RUN   TestAccDcsInstancesV1_basic
=== PAUSE TestAccDcsInstancesV1_basic
=== CONT  TestAccDcsInstancesV1_basic
--- PASS: TestAccDcsInstancesV1_basic (107.52s)

PASS
```

### CODE REVIEW

**Review by vladimirvshivkov — DISMISSED** [review]

**Review by artem-lifshits — CHANGES_REQUESTED** [review]
Tests are failing for me:
=== RUN   TestAccDcsInstancesV1_basic
=== PAUSE TestAccDcsInstancesV1_basic
=== CONT  TestAccDcsInstancesV1_basic
=== CONT  TestAccDcsInstancesV1_basic
    resource_opentelekomcloud_dcs_instance_v1_test.go:24: Step 1/2 error: After applying this test step, the plan was not empty.
        stdout:
        
        
        Terraform used the selected providers to generate the following execution
        plan. Resource actions are indicated with the following symbols:
        -/+ destroy and then create replacement
        
        Terraform will perform the following actions:
        
          # opentelekomcloud_dcs_instance_v1.instance_1 must be replaced
        -/+ resource "opentelekomcloud_dcs_instance_v1" "instance_1" {
              + access_user         = (known after apply)
              ~ capacity            = 0 -> 0.125 # forces replacement
              ~ created_at          = "2022-07-29T11:09:12.225Z" -> (known after apply)
              + description         = (known after apply)
              ~ id                  = "3efae1b9-3b30-4d5c-91db-f2a86cf2ede5" -> (known after apply)
              + internal_version    = (known after apply)
              ~ ip                  = "192.168.1.229" -> (known after apply)
              ~ maintain_begin      = "02:00:00" -> (known after apply)
              ~ maintain_end        = "06:00:00" -> (known after apply)
              ~ max_memory          = 128 -> (known after apply)
                name                = "dcs_instance_0apw0"
              + order_id            = (known after apply)
              ~ port                = 6379 -> (known after apply)
              ~ resource_spec_code  = "redis.ha.xu1.tiny.r2.128" -> (known after apply)
              + security_group_id   = "738517ba-a383-4653-a5c9-4d225ee5736e"
              + security_group_name = (known after apply)
              ~ status              = "RUNNING" -> (known after apply)
              ~ subnet_name         = "subnet-testing" -> (known after apply)
              ~ used_memory         = 2 -> (known after apply)
              ~ user_id             = "087482cbabbc4d58890d2cfad91afd29" -> (known after apply)
              ~ user_name           = "alifshit" -> (known after apply)
              ~ vpc_name            = "vpc-testing" -> (known after apply)
                # (7 unchanged attributes hidden)
        
        
                # (2 unchanged blocks hidden)
            }
        
        Plan: 1 to add, 0 to change, 1 to destroy.
--- FAIL: TestAccDcsInstancesV1_basic (84.44s)

@anton-sidelnikov @vladimirvshivkov can you also check? Thanks

**Comment by vladimirvshivkov** [comment]
Fails for me too.
But I cant even create dcs in web console, I guess there is some quota issue.

**Comment by Aloento** [comment]
@artem-lifshits @anton-sidelnikov could you run the test again? Thanks.

**Review by artem-lifshits — DISMISSED** [review]

File: releasenotes/notes/change-capacity-to-float32-2ac71ac93c328888.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    **[DCS]** Change Capacity from int to float32 (`#1842 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/1842>`_)
```
Comment (vladimirvshivkov) [inline releasenotes/notes/change-capacity-to-float32-2ac71ac93c328888.yaml]: float64

**Comment by vladimirvshivkov** [comment]
```
=== RUN   TestAccDcsAZV1DataSource_basic
=== PAUSE TestAccDcsAZV1DataSource_basic
=== CONT  TestAccDcsAZV1DataSource_basic
--- PASS: TestAccDcsAZV1DataSource_basic (34.52s)
=== RUN   TestAccDcsMaintainWindowV1DataSource_basic
=== PAUSE TestAccDcsMaintainWindowV1DataSource_basic
=== CONT  TestAccDcsMaintainWindowV1DataSource_basic
--- PASS: TestAccDcsMaintainWindowV1DataSource_basic (34.45s)
=== RUN   TestAccDcsProductV1DataSource_basic
=== PAUSE TestAccDcsProductV1DataSource_basic
=== CONT  TestAccDcsProductV1DataSource_basic
--- PASS: TestAccDcsProductV1DataSource_basic (35.24s)
=== RUN   TestAccDcsInstancesV1_basic
=== PAUSE TestAccDcsInstancesV1_basic
=== CONT  TestAccDcsInstancesV1_basic
--- PASS: TestAccDcsInstancesV1_basic (131.72s)
=== RUN   TestAccDcsInstancesV1_basicSingleInstance
=== PAUSE TestAccDcsInstancesV1_basicSingleInstance
=== CONT  TestAccDcsInstancesV1_basicSingleInstance
--- PASS: TestAccDcsInstancesV1_basicSingleInstance (73.84s)
PASS

Process finished with the exit code 0
```

**Review by vladimirvshivkov — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**